sdk/datafactory/arm-datafactory/package.json

```diff
@@ -29,12 +29,12 @@
   "types": "./types/arm-datafactory.d.ts",
   "devDependencies": {
     "@microsoft/api-extractor": "^7.18.11",
-    "@rollup/plugin-commonjs": "^21.0.1",
+    "@rollup/plugin-commonjs": "11.0.2",
```

Comment:
why do we have this change ? is it from code generator ?

Reply:
I have regenerated the package.